src/ert/_c_wrappers/job_queue/ext_job.py

```diff
@@ -33,6 +38,8 @@ class ExtJob:
     define_args: SubstitutionList = field(default_factory=SubstitutionList)
     help_text: str = ""
 
+    _SUBTITUTED_AT_EXECUTION_TIME: List[str] = field(default=("<ITER>", "<IENS>"))
```

Comment:
Since this is a dataclass `_SUBSTITUTED_AT_EXECUTION_TIME` becomes an input parameter to `__init__`. I suggest we make this a constant outside the class.

Reply:
ah right of course outside the class, in the file... thank you : )

Reply:
fixed.